src/ui/modules/VideoHero.tsx

```
@@ -0,0 +1,311 @@
+      {/* SEO-friendly metadata */}
+      <div className="hidden">
+        <h1>{data?.title || 'Video'}</h1>
+      </div>
+
+      {!isPlaying ? (
+        // Thumbnail view
+        <div
+          className="relative w-full h-full cursor-pointer bg-black"
+          onClick={handlePlayClick}
+          tabIndex={0}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)